test/search/test_organized.cpp

```diff
@@ -246,32 +243,16 @@ TEST (PCL, Organized_Neighbor_Pointcloud_Neighbours_Within_Radius_Search)
     organizedNeighborSearch.setInputCloud (cloudIn);
     organizedNeighborSearch.radiusSearch (searchPoint, searchRadius, cloudNWRSearch, cloudNWRRadius);
 
-    // check if result from organized radius search can be also found in bruteforce search
+    // check if results from organized radius search are indeed within the search radius
     for (const auto& current : cloudNWRSearch)
     {
-      pointDist = std::sqrt (
-          ((*cloudIn)[current].x-searchPoint.x) * ((*cloudIn)[current].x-searchPoint.x) +
-          ((*cloudIn)[current].y-searchPoint.y) * ((*cloudIn)[current].y-searchPoint.y) +
-          ((*cloudIn)[current].z-searchPoint.z) * ((*cloudIn)[current].z-searchPoint.z)
-      );
-
-      ASSERT_LE (pointDist, searchRadius);
-    }
-
-
-    // check if bruteforce result from organized radius search can be also found in bruteforce search
-    for (const auto& current : cloudSearchBruteforce)
```

Comment:
Note: this loop does not do anything useful, it basically just does the brute force radius search from above again. So I removed it (and two similar loops in test_organized_index.cpp)